<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlareProperty</class>
 <widget class="QWidget" name="FlareProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{
	background-color:transparent;
    border-bottom-left-radius: 3px;
    border-bottom-right-radius: 3px;
    border: 1px solid #005192;
    border-top: none;
    margin-top: 18px;
}
 
QGroupBox::title
{
	subcontrol-origin: margin;	
    background:url(:/assets/subWindowBackground);
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    border: 1px solid #005192;
    border-bottom: none;
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #005192, stop: 1 #002f60);
    padding: 2 5000 2 2; /* some large value, as QGrouBox actual width is not known */
    color: white; 
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="ScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>327</width>
        <height>684</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="4" column="0">
        <widget class="GroupBox" name="groupBox_4">
         <property name="title">
          <string>Map</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="PictureBox" name="Map">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>160</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="filter">
             <string>Texture files(*.dds)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="GroupBox" name="groupBox_11">
         <property name="title">
          <string>Occluder</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="1">
           <widget class="SpinBox" name="VisiblePixelOffset">
            <property name="maximum">
             <number>100000</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_162">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Visible pixel offset:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="DoubleSpinBox" name="VisibleAngleOffset">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_163">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Visible angle offset:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_165">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Position offset Y:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="DoubleSpinBox" name="PositionOffsetY">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="DoubleSpinBox" name="PositionOffsetX">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_164">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Position offset X:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="DoubleSpinBox" name="PositionOffsetZ">
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_166">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Position offset Z:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="GroupBox" name="groupBox_2">
         <property name="title">
          <string>Flare</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="Enable">
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ColorSelector" name="BoundColor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="color" stdset="0">
             <color>
              <red>255</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="6">
        <widget class="GroupBox" name="groupBox">
         <property name="title">
          <string>Flares</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="ToolBarLayout">
            <property name="spacing">
             <number>3</number>
            </property>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="ListWidget" name="FlareList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>111</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::AllEditTriggers</set>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="GroupBox" name="groupBox_3">
         <property name="title">
          <string>Flares properties</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_161">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Diffuse:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_157">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Layer:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="ColorSelector" name="DiffuseColor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="DoubleSpinBox" name="DiffuseIntensity">
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_160">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Scale mulY:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="AutoRotate">
            <property name="text">
             <string>Auto rotate</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="AutoScale">
            <property name="text">
             <string>Auto scale</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_159">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Scale mulX:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="DoubleSpinBox" name="ScaleMulX">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="minimum">
             <double>-999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_156">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Rotation:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="DoubleSpinBox" name="ScaleMulY">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="minimum">
             <double>-999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="DoubleSpinBox" name="Rotation">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="DoubleSpinBox" name="Layer">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>2048.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="DoubleSpinBox" name="PositionMul">
            <property name="decimals">
             <number>5</number>
            </property>
            <property name="minimum">
             <double>-999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_158">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Position mul:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="FlareDuplicate">
   <property name="icon">
    <iconset resource="geditor.qrc">
     <normaloff>:/Resources/MetroStyle/clone-24.png</normaloff>:/Resources/MetroStyle/clone-24.png</iconset>
   </property>
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="toolTip">
    <string>Duplicate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="FlareCopy">
   <property name="icon">
    <iconset resource="geditor.qrc">
     <normaloff>:/Resources/MetroStyle/copy.png</normaloff>:/Resources/MetroStyle/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="FlarePaste">
   <property name="icon">
    <iconset resource="geditor.qrc">
     <normaloff>:/Resources/MetroStyle/cllipboard.png</normaloff>:/Resources/MetroStyle/cllipboard.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="FlareAdd">
   <property name="icon">
    <iconset resource="geditor.qrc">
     <normaloff>:/Resources/MetroStyle/plus.png</normaloff>:/Resources/MetroStyle/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="FlareRemove">
   <property name="icon">
    <iconset resource="geditor.qrc">
     <normaloff>:/Resources/MetroStyle/minus.png</normaloff>:/Resources/MetroStyle/minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="FlareRename">
   <property name="icon">
    <iconset resource="geditor.qrc">
     <normaloff>:/Resources/MetroStyle/rename-26.png</normaloff>:/Resources/MetroStyle/rename-26.png</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Rename</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QLabel</extends>
   <header>PictureBox.h</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>DoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>SpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>ScrollArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPreview</class>
   <extends>QWidget</extends>
   <header>ColorPreview.h</header>
  </customwidget>
  <customwidget>
   <class>ColorSelector</class>
   <extends>ColorPreview</extends>
   <header>ColorSelector.h</header>
  </customwidget>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>GroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>ListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="geditor.qrc"/>
 </resources>
 <connections/>
</ui>
